### PR TITLE
Sync to db cli

### PR DESCRIPTION
**Pull request by ArgentFalcon**
Fix for the sync to db code that now has it's own CLI application

### CODE REVIEW

**Comment by ArgentFalcon** [comment]
👀 @youngyjd @sherryzhao1006 
cc @lyft/dp-tools-viz

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -1842,6 +1842,13 @@ def sync_perm(args): # noqa
         print('The sync_perm command only works for rbac UI.')
 
 
+@cli_utils.action_logging
+def sync_to_db(args): # noqa
+    dagbag = DagBag(settings.DAGS_FOLDER, store_serialized_dags=settings.STORE_SERIALIZED_DAGS)
```
Comment (mik-laj) [inline airflow/bin/cli.py:1847]: Does Dagbag have sync_to_db method? In Airflow 2.0, we have it. In Airflow 1.10 it doesn't exist yet. How is it in this case?

Comment (ArgentFalcon) [reply airflow/bin/cli.py:1847]: doesn't exist yet. I could port it over as well, but it does basically the same thing.

File: tests/cli/test_cli.py [hunk 2]
```diff
@@ -380,6 +380,22 @@ def test_backfill(self, mock_run):
         )
         mock_run.reset_mock()
 
+    # @mock.patch("airflow.bin.cli.DAG.sync_to_db")
```
Comment (milton0825) [inline tests/cli/test_cli.py:383]:
```suggestion
    
```

File: tests/cli/test_cli.py [hunk 3]
```diff
@@ -380,6 +380,22 @@ def test_backfill(self, mock_run):
...
+    
+    @mock.patch.object(DAG, "sync_to_db", autospec=True)
+    def test_cli_sync_to_db(self, mock_sync):
+        all_dags_called = []
+
+        def sync_to_db(self):
+            all_dags_called.append(self.dag_id)
+
+        mock_sync.side_effect = sync_to_db
+        env = os.environ.copy()
```
Comment (milton0825) [inline tests/cli/test_cli.py:392]:
```suggestion

```

**Comment by astahlman** [comment]
👍

**Comment by sherryzhao1006** [comment]
👍

**Pull request merged**